substrate/frame/examples/src/lib.rs

```diff
@@ -38,3 +38,6 @@
 //!
 //! - [**`pallet-example-split`**](./split): A simple example of a FRAME pallet demonstrating the
 //!   ability to split sections across multiple files.
+//!
+//! - [**`pallet-example-storage-migrations`**](./storage-migrations): An example pallet
```

Comment:
I think this should work
```suggestion
//! - [`pallet-example-storage-migrations`]: An example pallet
```
It is the approach taken here https://github.com/paritytech/polkadot-sdk/pull/1546
These two prs might have some conflicts btw

Reply:
Strange rust docs behaviour: this worked fine when built locally, but links don't work on the deployed docs site. 

Locally:
<img width="1134" alt="Screenshot 2023-09-19 at 11 58 13 AM" src="https://github.com/paritytech/polkadot-sdk/assets/23283108/dbad4406-2c8b-48e3-b000-487569c9c974">

On deployed site:
<img width="1134" alt="Screenshot 2023-09-19 at 11 58 07 AM" src="https://github.com/paritytech/polkadot-sdk/assets/23283108/7d7f5388-b658-402c-bf6e-329b167bac9a">

Not sure how to fix this other than linking to the actual url of each example pallet crate. Related issue I found: https://github.com/rust-lang/docs.rs/issues/204

Reply:
I'll try fix it in this PR.